frontend/src/pages/settings/server.vue

```diff
@@ -0,0 +1,255 @@
+    </template>
+
+    <template #content>
+      <VCol
+        md="6"
+        class="uno-pb-4 uno-pt-0">
+        <h3 class="mb-2 text-lg font-bold">
+          General Server Settings
+        </h3>
+
+        <VTextField
+          v-model="serverSettings.state.value.name"
+          label="Server Name" />
+
+        <VSelect
+          v-model="serverSettings.state.value.language"
+          :loading="saving"
+          :disabled="saving"
+          variant="outlined"
+          :label="$t('preferredLanguage')"
+          :rules="SomeItemSelectedRule"
+          item-title="Name"
+          item-value="Value"
+          :items="culturesList" />
+
+        <VCheckbox
+          v-model="serverSettings.state.value.quickConnect"
+          label="Enable Quick Connect" />
+
+        <h3 class="text-lg font-bold mt-6 mb-2">
+          Paths
+        </h3>
+
+        <VTextField
+          v-model="serverSettings.state.value.cachePath"
+          label="Cache Path" />
+
+        <VTextField
+          v-model="serverSettings.state.value.metadataPath"
+          label="Metadata Path" />
+
+        <h3 class="text-lg font-bold mt-6 mb-2">
+          Branding
+        </h3>
+
+        <VTextField
+          v-model="serverSettings.state.value.loginDisclaimer"
+          label="Login Disclaimer" />
+
+        <VTextField
+          v-model="cssPreview"
+          label="Custom CSS"
+          readonly
+          @click="openCssDialog" />
+
+        <VCheckbox
+          v-model="serverSettings.state.value.enableSplash"
+          label="Enable Splash Screen" />
+
+        <h3 class="text-lg font-bold mt-6 mb-2">
+          Performance
+        </h3>
+
+        <VTextField
+          v-model.number="serverSettings.state.value.parallelLibraryScan"
+          label="Parallel Library Scan Limit"
+          type="number" />
+
+        <VTextField
+          v-model.number="serverSettings.state.value.parallelImageEncoding"
+          label="Parallel Image Encoding Limit"
+          type="number" />
+
+        <VBtn
+          variant="flat"
+          width="8em"
+          color="primary"
+          :loading="loading"
+          @click="saveSettings">
+          {{ $t('save') }}
+        </VBtn>
+      </VCol>
+
+      <VDialog
+        v-model="cssDialog"
+        persistent
+        max-width="600px">
+        <VCard>
+          <VCardTitle>Edit Custom CSS</VCardTitle>
+          <VCardText>
+            <VTextarea
+              v-model="serverSettings.state.value.customCSS"
+              rows="10"
+              auto-grow
+              outlined />
+          </VCardText>
+          <VCardActions>
+            <VSpacer />
+            <VBtn
+              text
+              @click="cssDialog = false">
+              Cancel
+            </VBtn>
+            <VBtn
+              color="primary"
+              @click="cssDialog = false">
+              OK
+            </VBtn>
+          </VCardActions>
+        </VCard>
+      </VDialog>
```

Comment:
This is a popup window to add the css onto.

![image](https://github.com/user-attachments/assets/784268d0-d5c4-4674-bc44-88aca75dd832)